src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -305,6 +305,36 @@ if [ ! -z ${RunCrossGen+x} ]%3B then
   ReleaseLock
 fi
 
+$(BashLinkerTestCleanupCmds)
+      ]]></BashCLRTestLaunchCmds>
+      <BashCLRTestLaunchCmds Condition="'$(CLRTestKind)' == 'BuildAndRun' And $(TargetOS) != 'Browser' And '$(TargetOS)' != 'Android' And '$(TargetArchitecture)' == 'arm64'">
+    <![CDATA[
+$(BashLinkerTestLaunchCmds)
+
+if [ ! -z "$CLRCustomTestLauncher" ];
+then
+    LAUNCHER="$CLRCustomTestLauncher $PWD/"
+else
+    LAUNCHER="$_DebuggerFullPath $(_CLRTestRunFile)"
+fi
+
+$(BashIlrtTestLaunchCmds)
+
+if [ ! -z ${RunCrossGen+x} ]%3B then
+  TakeLock
+fi
+
+export MONO_ENV_OPTIONS=--llvm
```

Comment:
unintentional?

Reply:
This is done intentionally, so that for LLVM AOT arm64 lane, it falls back to LLVM JIT.

Reply:
why would an AOT lane use the JIT?

Also if a CI lane needs a particular configuration, it should set some properties and then this file should set MONO_ENV_OPTIONS based on the value of the property.

Reply:
Oh wait, this is new stuff, not modifications of an existing configuration.
Nevermind.